ui/packages/catalog-ui-search/src/main/webapp/js/model/Workspace.js

```diff
@@ -104,7 +104,7 @@ module.exports = Backbone.AssociatedModel.extend({
     handleChange: function (model) {
         if (model !== undefined &&
             _.intersection(Object.keys(model.changedAttributes()), [
-                'result', 'saved', 'metacard.modified', 'id', 'subscribed'
+                'result', 'saved', 'metacard.modified', 'id', 'subscribed', 'serverPageIndex'
```

Comment:
How does this work?  It is almost backwards of what I was expecting... 

I am reading it as saying if any of these attributes are listed as changedAttributes... then the workspace is dirty and needs to be saved.  This seems like it would make serverPageIndex one of the attributes we need to check tp determine it is unsaved.  What am I missing?

Reply:
I agree, it's confusing, but what is actually happening here is that we're checking if the intersection of the sets of `changedAttributes` and the one of the known events we provide is empty. Meaning that a changed attribute falls outside of the expected one and that's when we set the workspaces as not saved.

Reply:
yeah... I am still confused.  I am thinking if the resultant list is 0... we are considering it dirty right, and needs saving?  And intersection is requiring it to be in both lists right?  

So, if changedAttributes is ['serverPageIndex', 'foo'], we set saved to false, if changedAttributes is ['foo'] we set saved to false, and if it is ['serverPageIndex'] then we don't change it.

Reply:
Yes, that is how it currently would work.

Reply:
Might be worth pulling this into a method to describe what it's doing, or pulling the array out into a constant.